lib/smart_answer/helpers/format_capture_helper.rb

```diff
@@ -0,0 +1,64 @@
+module SmartAnswer
+  module Helpers::FormatCaptureHelper
+    DEFAULT_FORMATS = {
+      govspeak: [/^body$/, /^post_body$/, /^next_steps$/],
+      text: [/^title$/, /^meta_description$/, /^hint$/, /^label$/, /^suffix_label$/, /^error_*./],
+    }.freeze
```

Comment:
Do you mean explicitly stating the format on `content_for :body, format: :html` calls in the template?

Reply:
Yup - though I'm a little biased because I quite liked the approach I used in https://github.com/alphagov/smart-answers/commit/92a14e31ea0ae5b297a3ffd96ee9a91448ca2315#diff-2079a30c43bb35352ff86cd242687980R25-R39 but that's probably just me being self satisfied with my own ideas.

Reply:
What sort of cryptic side effects could you see happening?

Overall, I was trying limit breaking changes to how templates are written. I'm keen to not add the burden of having to know what format a particular key should be when someone writes a template. Especially when certain keys must have a specific format e.g. :title is always :text. I was thinking because this behaviour is documented it should be clear what's going on. 

Also if you don't think block's this PR, we could create an issue for and revisit later?

Reply:
Cryptic for me would be that code that appears quite similar has different effects. For example:

```
<% content_for :body do %>
  **Bold text**
<% end %>
```

produces a different outcome to:

```
<% content_for :label do %>
  **Bold text**
<% end %>
```
---

Your point makes sense about not wanting to change too much and it's not something I think is blocking. Though I do think you still get advantages when for a field like title that can only be text because the code is also telling you it can only be text. If you take an example of say an error message that you want to add a link or emphasis too you then need to know where to look up the output since the code doesn't tell you what it's going to do.

Reply:
Captured in #4510